app/controllers/api/base_controller/manager.rb

```diff
@@ -2,8 +2,10 @@ module Api
   class BaseController
     module Manager
       def update_collection(type, id)
-        if @req.method == :put || @req.method == :patch
-          return send("#{@req.method}_resource", type, id)
+        if @req.method == :put
+          return edit_resource(type, id, @req.json_body)
+        elsif @req.method == :patch
+          return patch_resource(type, id)
         end
```

Comment:
Would it be cleaner to move to a case statement?

```suggestion
        case @req.method
        when :put
          return edit_resource(type, id, @req.json_body)
        when :patch
          return patch_resource(type, id)
        end
```